### PR TITLE
New travel advice edition has no default update type

### PR DESCRIPTION
**Pull request by laurentqro**
At the moment the "significant change" radio button is pre-checked, which probably encourages publishers to treat this as the default.

We should make it unchecked by default, as per [the design system's guidance on radios](https://design-system.service.gov.uk/components/radios/).

https://trello.com/c/LWl84KOC/336-dont-pre-check-a-significant-change-for-example-a-new-travel-restriction-when-publishers-create-new-editions

## Before

Screenshot 2020-06-17 at 17 07 39

## After

Screenshot 2020-06-25 at 09 59 38

Thank you your help @vanitabarrett ! 🎉

### CODE REVIEW

**Comment by chao-xian** [comment]
Think it might be worth updating this test to see if it still works? https://github.com/alphagov/travel-advice-publisher/blob/master/spec/features/edition_edit_spec.rb#L389

File: app/models/travel_advice_edition.rb [hunk 1]
```diff
@@ -150,6 +151,10 @@ def update?
 
 private
 
+  def update_type_is_specified
+    errors.add(:update_type, "can't be blank on publish") if minor_update.nil?
```
Comment (vanitabarrett) [inline app/models/travel_advice_edition.rb:155]: The new error message isn't very visible. Other error states have a message at the top but also highlight the field that's erroring. I wonder if we can add something like that?

Screenshot 2020-06-19 at 10 09 12

File: app/views/admin/editions/_change_notes.html.erb [hunk 2]
```diff
@@ -25,8 +25,8 @@
                       required: true,
                       class: "form-control",
                       disabled: ! @edition.draft?) %>
-    </div>
 
-    <p>Read the <a href="https://www.gov.uk/guidance/content-design/writing-for-gov-uk#change-notes" target="_blank">guidance about change notes</a> (opens in a new tab).</p>
+      <p>Read the <a href="https://www.gov.uk/guidance/content-design/writing-for-gov-uk#change-notes" target="_blank">guidance about change notes</a> (opens in a new tab).</p>
```
[the remark this answers is not in the document]
Comment (laurentqro) [reply app/views/admin/editions/_change_notes.html.erb]: We have control over what we're linking to here, so I think it should be ok @vanitabarrett ?

**Comment by vanitabarrett** [comment]
Code changes all look good to me, nice work 👍 

Could we tidy up the commits a bit? The first commit title "Make minor update the default for a new edition" doesn't properly reflect the change we're making - it's not minor by default, it doesn't have a default 🙂 And we may be able to squash some of the commits together.

**Pull request merged**